pnpm-workspace.yaml

```diff
@@ -137,7 +137,7 @@ catalog:
   tsup: ^8.3.5
   tsx: ^4.19.2
   typescript: ^5.7.3
-  unbuild: ^3.3.1
+  unbuild: 3.1.0
```

Comment:
We do need to pin it? Or should we fix upstream directly?

Reply:
The PR (https://github.com/unjs/unbuild/pull/475) is there, I only need to push my changes to include `export { default } from '<identifier>'` fix.

I will try to remove the cjs plugin included with version 3.3.1 in the afternoon:
- https://github.com/unjs/unbuild/blob/main/src/builders/rollup/build.ts#L85
- https://github.com/unjs/unbuild/blob/main/src/builders/rollup/plugins/cjs.ts#L18

Reply:
Oh, awesome! How about we do a patch from your PR with the latest version?

Reply:
In the meantime, I'm going to submit another pr to fix cjs exports temporarily.

Reply:
@antfu @hannoeru new PR here https://github.com/unocss/unocss/pull/4401